.github/workflows/rebuilder.py

```diff
@@ -3,13 +3,66 @@
 import json
 import logging
 import re
+import sys
 from typing import Set
 
 import copr.v3
 import dnf
 import hawkey
 
 
+class CoprBuild:
+
+    def __init__(self, data: dict):
+        self.data = data
+
+    @property
+    def id(self):
+        return self.data["id"]
+
+    @property
+    def state(self):
+        return self.data["state"]
+
+    @property
+    def submitted_on(self):
+        return self.data["submitted_on"]
+
+
+class CoprPkg:
+
+    def __init__(self, data: dict):
+        self.data = data
```

Comment:
I'm sorry for nitpicking but it is really hard to understand what needs to go into `data`. From looking in the code an element from the response of this [`copr_client.package_proxy.get_list`](https://python-copr.readthedocs.io/en/latest/client_v3/proxies.html#copr.v3.proxies.package.PackageProxy.get_list) method, needs to go here? The HTTP API response is documented [here](https://copr.fedorainfracloud.org/api_3/docs).